tests/test_minuit.py

```diff
@@ -213,6 +215,17 @@ def no_signature(*args):
     assert m.values == (3, 4)
     assert m.parameters == ("x0", "x1")
 
+    partial_no_signature = partial(no_signature, 42)
+    m = Minuit(partial_no_signature, 3)
+    assert m.values == (3,)
+    assert m.parameters == ("x0",)
+    m.migrad()
+
+    m = Minuit(lambda *args: no_signature(42, *args), 2)
+    assert m.values == (2,)
+    assert m.parameters == ("x0",)
+    m.migrad()
+
```

Comment:
Is this second test really necessary? It seems to me that test_generic_partial() is sufficient. Am I missing something?

Reply:
It probably is. I first added this test, as the lines above and below already suggested a similar scenario and later realized that `tests/test_describe.py` is probably more fitting. I can remove it, sure. But I think it is also no problem if we keep it here. What do you think?

Reply:
I prefer to remove these extra tests and only keep the test_generic_partial, then. Could you make the change?

Reply:
Done!